docs: cover statsd metrics in extensions (#887)
canonical/rockcraft: docs/reference/extensions/django-framework.rst

         # list required packages or slices for your Django application below.
         - libpq-dev

StatsD exporter
===============

> A StatsD exporter is installed alongside the Gunicorn server to export Gunicorn
> server metrics. Three of the `Gunicorn-provided metrics

Reviewer: ```suggestion
A StatsD exporter is installed alongside the Gunicorn server to record
server metrics. Three of the `Gunicorn-provided metrics
```

<https://docs.gunicorn.org/en/stable/instrumentation.html>`_

> are mapped to new names: ``django_response_code``, ``django_requests`` and
> ``django_request_duration``.

Reviewer: I'd make a table for these.

```suggestion
are mapped to new names:

.. list-table::
    * - StatsD metric
       - Gunicorn metric
    * - ?
       - ``django_response_code``
    * - ``gunicorn.requests``
       - ``django_requests``
    * - ``gunicorn.request.duration``
       - ``django_request_duration``
```

I couldn't see what the response code one maps to.

> The  exporter listens on localhost at port 9125. You can push your
> own metrics to the statsd-exporter using any StatsD Client. For example,
> using the client ``pystatsd``:

Reviewer: ```suggestion
The  exporter listens on localhost at port 9125. You can push your
own metrics to the exporter using any StatsD client. This snippet from an example
Django app uses pystatsd as a client:
```

.. code-block:: python

  import statsd
  c = statsd.StatsClient('localhost', 9125)

>   c.incr('my_counter')

Reviewer 2: No, there's no particular file where the user needs to add this code block. It's just an example of what the user would need to add

         # list required packages or slices for your Django application below.
         - libpq-dev

StatsD exporter
===============

A StatsD exporter is installed alongside the Gunicorn server to record
server metrics. Some of the `Gunicorn-provided metrics
<https://docs.gunicorn.org/en/stable/instrumentation.html>`_
are mapped to new names:

.. list-table::

>   * - StatsD metric
>     - Gunicorn metric

Reviewer: I reversed these by accident.

```suggestion
  * - Gunicorn metric
    - StatsD metric
```
